Artifacts/windows-install-windows-updates/artifact.ps1

```diff
@@ -13,13 +13,11 @@ Push-Location $PSScriptRoot
 
 ###################################################################################################
 
-trap
-{
+trap {
```

Comment:
Is that the standard we use across the repo?

Reply:
Spot checked a few files and most have the braces on separate line.

Reply:
VS code formatter defaults.
Changed settings to have it on a separate line.

we might want to consider having a .editorconfig, to ensure the same rules are applied automatically.